paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -329,7 +329,7 @@ public class PaparazziPlugin : Plugin<Project> {
 
   private fun Project.addTestDependency() {
     val dependency = if (isInternal()) {
-      dependencies.project(mapOf("path" to ":paparazzi"))
+      dependencies.project(mapOf("path" to ":paparazzi-junit4"))
```

Comment:
deliberate for now.  can query gradle extension config for junit4 vs junit5, but not sure how to keep it generic for other 3rd party wrappers.